src/api/entities/Asset/Fungible/Issuance/index.ts

```diff
@@ -27,5 +25,5 @@ export class Issuance extends Namespace<FungibleAsset> {
    *
    * @param args.amount - amount of Asset tokens to be issued
```

Comment:
I think this comment should mention `portfolioId` is optional here
```suggestion
   * @param args.amount - amount of Asset tokens to be issued
   * @param args.portfolioId - ID of the portfolio to hold the issued tokens (optional)
```

Reply:
it was already added in the interface